receiver/kubeletstatsreceiver/README.md

```diff
@@ -218,6 +218,41 @@ receivers:
       - pod
 ```
 
+### Collect k8s.container.cpu.node_limit_utilization as ratio of total node's capacity
+
+In order to calculate the `k8s.container.cpu.node_limit_utilization` metric, the information of the node's capacity
+must be retrieved from the k8s API. In this, the `k8s_api_config` needs to be set.
+In addition, the node name must be identified properly. When the collector runs as Daemonset,
+the receiver can get this information automatically by the `K8S_NODE_NAME` env var:
```

Comment:
Do we need to explicitly call out that this metric should not be used in non-daemonset deployments?

Reply:
Well, the idea here is that the recommended way to run the `kubeletstats` receiver is  via a Daemonset. That's what the Helm presets also do. So in that case each Daemonset Pod is responsible for scraping its local Nod's endpoint. 

But at the same time we cannot prevent users from using a single Deployment Pod to scrape from all the cluster's nodes. In that case, I think the suggested way would be to use the `k8s_observer` in order to discover the Nodes and initialize one receiver for each one of them. In this scenario the `node` setting should be populated by the name of each discovered Node accordingly and not by `K8S_NODE_NAME` env var which actually would point to the local Node. That is what I'm trying to illustrate with the provided configuration bellow which uses the `k8s_observer` approach.
Let me know if that makes sense.

Reply:
Can you include in the README an example of setting `node` with the env var and enabling the metric?

Reply:
Added.